ui/app/components/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -38,11 +38,17 @@ export default class SenderToRecipient extends PureComponent {
     recipientAddressCopied: false,
   }
 
```

Comment:
Are any of the changes in this file required for the fix? Or is this just a refactor?

Reply:
The two separate commits here are independent, yup. The changes to this file (the first commit) aren't needed per se, but do improve the component. The 2nd commit with the fix doesn't touch this file, no.